doc/source/User_guide/pyaedt_file_data/project.rst

```diff
@@ -0,0 +1,159 @@
+                "Include Temperature": true,
+            }
+        },
+        "boundaries": {
+            # Boundary Name : {Boundary Properties}
+            "CPU": {
+                "Objects": [
+                    "CPU"
+                ],
+                "Block Type": "Solid",
+                "Use External Conditions": false,
+                "Total Power": "25W",
+                "BoundType": "Block"
+            },
+        },
+        "mesh": {
+            "Settings": {
+                # mesh_properties,
+                "MeshMethod": "MesherHD",
+                "UserSpecifiedSettings": true,
+                "ComputeGap": true,
+                "MaxElementSizeX": "16mm",
+                "MaxElementSizeY": "3.5mm",
+                "MaxElementSizeZ": "11mm",
+                # ....
+            }
+        },
+        "materials": {
+            # Material Name : {Material Properties}
+            "Al-Extruded": {
+                "CoordinateSystemType": "Cartesian",
+                "BulkOrSurfaceType": 1,
+                "PhysicsTypes": {
+                    "set": [
+                        "Thermal"
+                    ]
+                },
+                "AttachedData": {
+                    "MatAppearanceData": {
+                        "property_data": "appearance_data",
+                        "Red": 232,
+                        "Green": 235,
+                        "Blue": 235
+                    }
+                },
+                "thermal_conductivity": "205",
+                "mass_density": "2800",
+                "specific_heat": "900",
+                "thermal_material_type": {
+                    "property_type": "ChoiceProperty",
+                    "Choice": "Solid"
+                },
+                "clarity_type": {
+                    "property_type": "ChoiceProperty",
+                    "Choice": "Opaque"
+                }
+            },
+        },
+        "objects": {
+            # Object Name: {object properties}
+            "Region": {
+                "SurfaceMaterial": "",
+                "Material": "air",
+                "SolveInside": true,
+                "Model": true,
+                "Group": "",
+                "Transparency": 0.0,
+                "Color": [
+                    255,
+                    0,
+                    0
+                ],
+                "CoordinateSystem": "Global"
+            },
+
+        },
+        "datasets": [
+            # Dataset Name : {Dataset Properties}
+        ],
+        "monitors": [
+            # Monitor Name : {Monitor Properties}
+    ],
+        "native components": {
+            # Component Name : {Component Properties}
+
+    }
+    }
+
+For a practical demonstration, refer to the provided example in the following link:
+`Project configuration file example <https://aedt.docs.pyansys.com/version/stable/examples/01-Modeling-Setup/Configurations.html#sphx-glr-examples-01-modeling-setup-configurations-py>`_
```

Comment:
```suggestion
For a practical demonstration, see the
`Project configuration file example <https://aedt.docs.pyansys.com/version/stable/examples/01-Modeling-Setup/Configurations.html#sphx-glr-examples-01-modeling-setup-configurations-py>`_
```
This seems to be a link to an example. The display text for the link should be the same as the title of the example.